src/main/java/vivatech/util/MachineTiers.java

```diff
@@ -0,0 +1,29 @@
+package vivatech.util;
+
+public enum MachineTiers {
+	
+	MINIMAL(1, 0, "_minimal"), NORMAL(2, 3, "_normal"), ADVANCED(3, 5, "_advanced");
```

Comment:
(see my other comment about underscores) Why not "primitive, industrial, advanced" ?

Reply:
I think this is simpler. That’s my idea